load.php

```diff
@@ -11,16 +11,17 @@
 require_once __DIR__ . '/inc/namespace.php';
 
 add_action( 'hm-platform.modules.init', function () {
+	$is_cloud = in_array( get_environment_architecture(), [ 'ec2', 'ecs' ], true );
 	$default_settings = [
-		'enabled'            => in_array( get_environment_architecture(), [ 'ec2', 'ecs', 'local-server' ] ),
+		'enabled'            => true,
 		'cavalcade'          => true,
 		's3-uploads'         => true,
-		'aws-ses-wp-mail'    => true,
-		'batcache'           => true,
+		'aws-ses-wp-mail'    => $is_cloud,
+		'batcache'           => $is_cloud,
 		'redis'              => true,
 		'ludicrousdb'        => true,
 		'healthcheck'        => true,
-		'xray'               => true,
+		'xray'               => $is_cloud,
```

Comment:
We use / support this in local-server, so might need a way to modify this one

Reply:
Interesting; using a local XRay, or pushing to live? We should try and support in Chassis if it's not too hard.

Reply:
The xray daemon runs locally but then pushed to aws. You need aws creds currently to do that. It's mostly used for developing the xray plugin its self.